test/tests/fvbcs/fv_functor_dirichlet/fv_other_side.i

```diff
@@ -0,0 +1,103 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '1 2'
+    dy = '1.3'
+    ix = '5 10'
+    iy = '3'
+    subdomain_id = '0 1'
+  []
+  [mid]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = 'cmg'
+    primary_block = '0'
+    paired_block = '1'
+    new_boundary = 'mid'
+  []
+[]
+
+[Variables]
+  [u]
+    type = MooseVariableFVReal
+    block = 0
+  []
+  [v]
+    type = MooseVariableFVReal
+    block = 1
+  []
+[]
+
+[FVKernels]
+  [diffu]
+    type = FVDiffusion
+    variable = u
+    coeff = 1
+  []
+  [diffv]
+    type = FVDiffusion
+    variable = v
+    coeff = 2
+    block = 1
+  []
+  [source]
+    type = FVBodyForce
+    variable = v
+    value = 1
+  []
+[]
+
+[FVBCs]
+  [left]
+    type = FVDirichletBC
+    variable = u
+    boundary = left
+    value = 4
+  []
+  [mid]
+    type = FVADFunctorDirichletBC
+    variable = u
+    functor = v
+    functor_only_defined_on_other_side = true
+    ghost_layers = 3
```

Comment:
This doesn't make sense. We should figure it out

Reply:
I can look into it if you would prefer

Reply:
do we need the point neighbor ghosting?
I was thinking we dont because the side values of the cell on the other side dont depend on the cells that only touch with a corner

Reply:
but if we did that would explain the "3"

Reply:
We should need one side neighbor layer directly from the FV Dirichlet BC, and then if the functor is a block restricted variable we will need one additional side neighbor layer to evaluate its cell gradient, so two side neighbor layers. In theory two *point* neighbor layers would be overkill and one *point* neighbor layer would be insufficient

Reply:
Never mind, I can see how we would get 3. A non-interface face needs needs neighboring cell gradients for the un-corrected face gradient (1). The cell gradient calls the FV Dirichlet BC which triggers the neighbor face evaluation (2). The neighbor face evaluation for two term expansion calculates a cell gradient which requires neighbor face values (3)